cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,32 @@
 /// <reference types='cypress' />
 
-describe('Book Store app', () => {
-  before(() => {
-    
+describe('Successful login', () => {
+  beforeEach(() => {
+    cy.visit('/login');
+    cy.get('#userName').type('haha.test11');
+    cy.get('#password').type('Passwprd!123');
+    cy.contains('.text-right button', 'Login').click();
+    cy.get('#userName-value').should('contain.text', '');
```

Comment:
Change the assertion to check for the actual username instead of an empty string.